package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "setup-dbs": "psql -f ./db/setup.sql",
-    "seed": "node ./db/seeds/run-seed.js",
-    "test": "jest",
+    "seed": "PGDATABASE=nc_news node ./db/seeds/run-seed.js",
```

Comment:
this needs to be removed. We should not be setting environment variables in your package.json